### DISCUSSION TITLE
Words in Italics in Biblical Text

### DISCUSSION THREAD

**Discussion by guiquimthe** [ezra-bible-app/ezra-bible-app#602] [Q&A] [answered]
Hello my brothers in Christ,

Good afternoon,

Does the Ezra Bible App not allow you to view the biblical text with the words in italics, which indicate words not found in the original Hebrew, nor in Aramaic, nor in Greek, but implied in the context of the verse?

The King James Version 1769 and other translations (in Portuguese, Almeida Corrigida and Fiel - acf) have used italics and this issue seems important to me.

Thanks for listening,

Guilherme

**Comment by tobias-klein** [comment, marked as answer]
Good question! I fear the rendering of that markup is currently not supported. I'll have a look and see what we can do about that! I am guessing that it would not be too much work to fix that!

**Reply by tobias-klein** [reply]
I just had a look at the KJV markup! It contains the `<transchange>` element which signifies what you mentioned.

I added a quick CSS change for a test and based on that the example section Ephesians 1:20-23 would look like this:
![image](https://user-images.githubusercontent.com/17739895/148115466-e734caea-3648-40be-8e1f-6fed2e422152.png)

And this is how [STEP Bible](https://www.stepbible.org/?q=reference=Eph.1|version=KJV&options=HVLGUN) formats it:
![image](https://user-images.githubusercontent.com/17739895/148115584-ae764895-16be-4167-ab29-ffc8596eed27.png)
They are using a different color for the regular text than for the `<transchange>` element.

We can certainly add a style for this markup. Would italic be sufficient for now? (We can always optimize it further in another step) Any additional feedback @guiquimthe?

**Reply by guiquimthe** [reply]
@tobias-klein I admire your readiness for the service. For me, it's enough to highlight this part. I believe I have no further questions about it, at least for now.
